README.md

```diff
@@ -60,120 +48,159 @@ does not support (directly), repository specific fields in all cases. It's basic
 
 Once you have that done you must set the following properties
 
 Then, you have to set some environment variables as this is how the acceptance tests pick up their config
-```bash
+```sh
 ARTIFACTORY_URL=http://localhost:8082
 ARTIFACTORY_USERNAME=admin
 ARTIFACTORY_PASSWORD=password
-TF_ACC=FOO
+TF_ACC=true
 ```
-a crucial, and very much hidden, env var to set is
-`TF_ACC=FOO` - you can literally set `TF_ACC` to anything you want, so long as it's set. The acceptance tests use
-terraform testing libraries that, if this flag isn't set, will skip all tests.
+A crucial, and very much hidden, env var to set is `TF_ACC=true` - you can literally set `TF_ACC` to anything you want, so long as it's set. The acceptance tests use terraform testing libraries that, if this flag isn't set, will skip all tests.
 
 You can then run the tests as
-`go test -v ./pkg/...`
-**DO NOT** remove the `-v` - terraform testing needs this (don't ask me why). This will recursively run all tests, including
-acceptance tests. 
+
+```sh
+$ go test -v ./pkg/...
+```
+
+Or
+
+```sh
+$ make acceptance
+```
+
+**DO NOT** remove the `-v` - terraform testing needs this (don't ask me why). This will recursively run all tests, including acceptance tests.
+
+### Testing Federated repos
+
+To execute acceptance tests for federated repos resource, we need:
+- 2 Artifactory instances, configured with Circle-of-Trust
+- Set environment variables `ARTIFACTORY_TEST_FEDERATED_REPO` to enable the acceptance tests that utilize both Artifactory instances
+
+#### Setup Artifactory instances
+
+Instead of using [scripts/run-artifactory.sh](scripts/run-artifactory.sh) to start one Artifactory instance for testing, use the file `scripts/docker-compose.yml` to startup *2* Artifactory instances.
+
+`artifactory-1` will be on the usual 8080/8081/8082 ports while `artifactory-2` is on 9080/9081/9082
+
+```sh
+$ docker-compose up -d
+```
+
+Use `docker-compose logs -f` to monitor startup progress.
+
+Once both instances are up and running (this will take up to a minute or more), set their base URLs using
+```sh
+$ curl -X PUT http://localhost:8081/artifactory/api/system/configuration/baseUrl -d 'http://artifactory-1:8081' -u admin:password -H "Content-type: text/plain"
+$ curl -X PUT http://localhost:9081/artifactory/api/system/configuration/baseUrl -d 'http://artifactory-2:8081' -u admin:password -H "Content-type: text/plain"
+```
+
+Setup [Circle-of-Trust](https://www.jfrog.com/confluence/display/JFROG/Access+Tokens#AccessTokens-CircleofTrust(Cross-InstanceAuthentication)) by copying each instance's `root.crt` to the other instance.
+
+Get the container IDs:
+```sh
+$ docker container ls
+```
+
+Get the `root.crt` from each instance
+```sh
+$ docker cp <container ID for artifactory-1>:/opt/jfrog/artifactory/var/etc/access/keys/root.crt artifactory-1.crt && chmod go+rw artifactory-1.crt
```

Comment:
```bash
docker cp "$(docker ps | grep dazzling_austin |\
     awk '{print $1}' ):/opt/jfrog/artifactory/var/etc/access/keys/root.crt" \
    && chmod go+rw artifactory-1.crt
```

I prefer working c&p-able code

Reply:
You can 1-line this or but it in a bash script - The best explanation is a script. Then it just works